help_to_heat/settings.py

```diff
@@ -258,7 +258,7 @@ def show_toolbar(request):
 
 TOTP_ISSUER = "Help to Heat Supplier Portal"
 
-CSRF_TRUSTED_ORIGINS = [BASE_URL]
+CSRF_TRUSTED_ORIGINS = ["https://*.check-eligibility-for-gb-insulation-scheme.service.gov.uk"]
```

Comment:
This is probably fine, but it would allow requests between environments, which isn't ideal. And it should probably come from config rather than being hard-coded.

If it was the trailing slash that was the problem, can we trim that from the base URL?